### PR TITLE
Support automatic labeling of kube volumes

### PR DESCRIPTION
**Pull request by rhatdan**
Allow users to specify options on the volume mount path.
This will trigger relabels of user specifies :z,:Z
Also will handle User Relabels if the user specifies :U

Fixes: https://github.com/containers/podman/issues/9371

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@rhatdan lint is complaining about trailing whitespace.

**Review by giuseppe — APPROVED** [review]
LGTM

File: test/system/700-play.bats [hunk 1]
```diff
@@ -51,18 +51,40 @@ spec:
...
 
+RELABEL="system_u:object_r:container_file_t:s0"
+
 @test "podman play with stdin" {
-    echo "$testYaml" > $PODMAN_TMPDIR/test.yaml
+    TESTDIR=$PODMAN_TMPDIR/testdir
+    mkdir -p $TESTDIR
+    echo "$testYaml" | sed "s|TESTDIR|${TESTDIR}|g" > $PODMAN_TMPDIR/test.yaml
     run_podman play kube - < $PODMAN_TMPDIR/test.yaml
+    if [ -e /usr/sbin/selinuxenabled -a /usr/sbin/selinuxenabled ]; then
```
Comment (edsantiago) [inline test/system/700-play.bats:73]: This does not do what you think it does (the `/usr/sbin/selinuxenabled` on the right-hand side is not executed, it is an empty-string check (equivalent to `test -n /usr/sbin/selinuxenabled`). I can fix this in a followup PR if you need to merge this urgently.

Comment (rhatdan) [reply test/system/700-play.bats:73]: We have this same check in other locations.  But let's fix it elsewhere so we can get this merged and move along.

**Comment by TomSweeneyRedHat** [comment]
LGTM
(and dang, @ashley-cui beat me to the shiny merge button once again!)

**Pull request merged**